mgmt/ProcessManager.cc

```diff
@@ -218,14 +218,14 @@ ProcessManager::reconfigure()
 void
 ProcessManager::signalConfigFileChild(const char *parent, const char *child, unsigned int options)
 {
-  static const MgmtMarshallType fields[] = {MGMT_MARSHALL_STRING, MGMT_MARSHALL_STRING, MGMT_MARSHALL_INT};
+  MgmtMarshallInt mgmtopt    = options;
+  MgmtMarshallString mparent = const_cast<MgmtMarshallString>(parent);
```

Comment:
It gives an assignment error from the compiler. I think `const MgmtMarshallString` gets interpreted as `char * const` since `MgmtMarshallString` is just an alias for `char *`.